config/rbac/service_account.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: deployer
+  namespace: system
```

Comment:
Why are creating the new service account in the `system` namespace? or is the make process patch the correct namespace here?

Reply:
Yes, the make process will substitute the correct namespace